.config/cargo_spellcheck.dic

```diff
@@ -0,0 +1,50 @@
+60
+
+WebAssembly
+Wasm
+validator
+ERC
+fuzzer
+SHA
+BLAKE2
+KECCAK
+ABI
+FFI
+UI
+crypto
+deploy
+layed
+dispatchable
+encodable
+decodable
+callee
+prefilled
```

Comment:
This is actually not a word: https://www.dict.cc/?s=pre-filled&failed_kw=prefilled

Reply:
I had looked it up in Merriam Webster earlier today:
https://www.merriam-webster.com/dictionary/prefilled